package.json

```diff
@@ -0,0 +1,108 @@
+{
+  "name": "redux-debounce-field",
+  "version": "1.0.0",
+  "main": "dist/cjs/redux-debounce-field.js",
+  "module": "dist/es/redux-debounce-field.js",
+  "types": "dist/es/index.d.ts",
+  "repository": "git@github.secureserver.net:pexux/ux-redux-debounce-field",
+  "author": "Jack Ellis <jack.m.ellis@hotmail.co.uk>",
+  "license": "UNLICENCED",
+  "publishConfig": {
+    "registry": "https://artifactory.ba.heg.com/api/npm/godaddy-emea/",
```

Comment:
Can you not only remove this but scrub the commit. Don't really want these urls in open source.